src/translations/tr/common.json

```diff
@@ -193,4 +193,5 @@
   "selectionLi2": "Make sure your motors spin up reliably on arm and in the correct direction",
   "selectionLi3": "Check motor temperatures after a short test flight and adjust tune if they are too hot",
   "selectionLinkText": "Show release notes on Github"
+
```

Comment:
Do not add changes in translation files - they will be overwritten by crowdin.
Generally, please do not add empty changes.